From 656dfcf370a75ef22962de11a23512e67210f44c Mon Sep 17 00:00:00 2001
From: Fabian Vogt <fabian@ritter-vogt.de>
Date: Fri, 30 May 2025 00:34:48 +0200
Subject: [PATCH] Revert "hdrcalibrator: install the image instead of including
 it in the module"

This reverts commit a7ae8cf2de28636179867206574ef4cda47b6a1e.
---
 hdrcalibrator/CMakeLists.txt | 3 ++-
 hdrcalibrator/ui/Main.qml    | 2 +-
 2 files changed, 3 insertions(+), 2 deletions(-)

Index: kscreen-6.4.0/hdrcalibrator/CMakeLists.txt
===================================================================
--- kscreen-6.4.0.orig/hdrcalibrator/CMakeLists.txt
+++ kscreen-6.4.0/hdrcalibrator/CMakeLists.txt
@@ -7,11 +7,11 @@ qt_add_qml_module(hdrcalibrator
     QML_FILES
         ui/Main.qml
     RESOURCES
+        ui/images/graz.png
         ui/images/plasma-symbolic.svg
     DEPENDENCIES
         QtQuick
 )
-install(FILES "ui/images/graz.png" DESTINATION "${KDE_INSTALL_DATADIR}/kscreen")
 
 qt6_generate_wayland_protocol_client_sources(hdrcalibrator
     FILES ${WaylandProtocols_DATADIR}/staging/color-management/color-management-v1.xml
Index: kscreen-6.4.0/hdrcalibrator/ui/Main.qml
===================================================================
--- kscreen-6.4.0.orig/hdrcalibrator/ui/Main.qml
+++ kscreen-6.4.0/hdrcalibrator/ui/Main.qml
@@ -204,7 +204,7 @@ Window {
                             Image {
                                 width: parent.width
                                 height: parent.height
-                                source: StandardPaths.locate(StandardPaths.GenericDataLocation, "kscreen/graz.png")
+                                source: "images/graz.png"
                                 fillMode: Image.PreserveAspectFit
                             }
                         }
